src/arch/InputHandler/InputHandler_DirectInput.cpp

```diff
@@ -723,11 +696,8 @@ InputHandler_DInput::UpdateBuffered(
 							up = JOY_AUX_3;
 							down = JOY_AUX_4;
 						} else
-							LOG->MapLog("unknown input",
-										"Controller '%s' is returning an "
-										"unknown joystick offset, %i",
-										device.m_sName.c_str(),
-										in.ofs);
+							Locator::getLogger()->trace("Controller '{}' is returning an unknown joystick offset, {}",
```

Comment:
We want to be careful converting MapLogs to traces due to their possibly high traffic usage. 
In this specific case, it might be better to make this a warn. If it crops up as an issue later, we could remove this log entirely or find a new solution.

Reply:
Fixed.